js/source/video_source.js

```diff
@@ -73,9 +77,9 @@ VideoSource.prototype = util.inherit(Evented, /** @lends VideoSource.prototype *
     roundZoom: true,
 
     /**
-     * Return the HTML video element.
+     * Returns the HTML `video` element.
      *
-     * @returns {Object}
+     * @returns {Node} The HTML `video` element.
```

Comment:
I think documentation.js plans to leverage Flow's type checker. The type Flow would use here is `HTMLVideoElement`.